api-scanner/src/test/java/net/corda/plugins/KotlinAnnotationsTest.java

```diff
@@ -43,6 +43,7 @@
 
     private static final String[] expectedClassWithJvmStaticFunctionCompanion = {
         "public static final class net.corda.example.HasJvmStaticFunction$Companion extends java.lang.Object",
+        "  public <init>(kotlin.jvm.internal.DefaultConstructorMarker)",
```

Comment:
This isn't a valid test-case - Kotlin should also be adding an `int` for the bitmask.
```java
public <init>(int, kotlin.jvm.internal.DefaultConstructorMarker)",
```